src/devices/develco.ts

```diff
@@ -673,6 +673,38 @@ const definitions: DefinitionWithExtend[] = [
         exposes: [e.occupancy(), e.battery_low()],
         extend: [develcoModernExtend.addCustomClusterManuSpecificDevelcoGenBasic(), develcoModernExtend.readGenBasicPrimaryVersions()],
     },
+    {
+        whiteLabel: [{vendor: 'Frient', model: 'MOSZB-153', description: 'Motion Sensor 2 Pet', fingerprint: [{modelID: 'MOSZB-153'}]}],
+        zigbeeModel: ['MOSZB-153'],
+        model: 'MOSZB-153',
+        vendor: 'Develco',
+        description: 'Motion Sensor 2 Pet',
+        fromZigbee: [fz.occupancy_with_timeout, fz.temperature, fz.illuminance, fz.battery, fz.ias_occupancy_alarm_1_with_timeout],
```

Comment:
```suggestion
        extend: [temperature(), illuminance(), battery(), iasZoneAlarm({zoneType: 'occupancy', alarmTimeout: true, zoneAttributes: ['alarm_1']})],
```

Can you try with this? Should be able to remove toZigbee/fromZigbee/exposes/endpoint/configure.